plasmapy/particles/particle_class.py

```diff
@@ -409,6 +380,70 @@ def __init__(
 
             categories.add(Element['category'])
 
+
+
+
+
+##########################
+
+def __init__(
```

Comment:
`__init__` should follow the class docstrings and its indentation needs to be fixed